speech/api/Recognize/Recognize.cs

```diff
@@ -511,11 +511,6 @@ static async Task<object> StreamingRecognizeAsync(string filePath)
         // [START speech_transcribe_streaming_mic]
         static async Task<object> StreamingMicRecognizeAsync(int seconds)
         {
-            if (NAudio.Wave.WaveIn.DeviceCount < 1)
```

Comment:
LGTM, on first read I assumed something changed in 1.9.0 that broke this logic. Did we want to explicitly remove the check for an available Mic / does this affect automation?

Reply:
The DeviceCount property no longer exists in 1.9.  I didn't want to change the logic, but I had to.